### ISSUE TITLE
Logo overlap in social card

### ISSUE THREAD

**Issue by ofek** [squidfunk/mkdocs-material#6475]
### Bug description

When a custom logo is set, the site logo overrides it:

![index](https://github.com/squidfunk/mkdocs-material/assets/9677399/7e39fdf9-96c7-4ba1-a604-4aa9090de5aa)

### Related links

https://squidfunk.github.io/mkdocs-material/plugins/social/#option.logo

### Reproduction

```
plugins:
  social:
    cards_layout_options:
      logo: docs/assets/images/logo.svg
```

### Steps to reproduce

https://github.com/pypa/hatch/blob/master/mkdocs.yml

### Browser

Chrome

### Before submitting

- [X] I have read and followed the [bug reporting guidelines](https://squidfunk.github.io/mkdocs-material/contributing/reporting-a-bug/).
- [X] I have attached links to [the documentation](https://squidfunk.github.io/mkdocs-material/), and possibly related [issues](https://github.com/squidfunk/mkdocs-material/issues) and [discussions](https://github.com/squidfunk/mkdocs-material/discussions).
- [X] I assure that I have [removed all customizations](https://squidfunk.github.io/mkdocs-material/contributing/reporting-a-bug/#remove-customizations) before submitting this bug report.
- [X] I have attached a __.zip file__ with a [minimal reproduction](https://squidfunk.github.io/mkdocs-material/guides/creating-a-reproduction/).

**Comment by squidfunk** [comment]
Thanks for reporting. A mere `mkdocs.yml` is not a self-contained reproduction. Please carefully read our [bug reporting guide](https://squidfunk.github.io/mkdocs-material/contributing/reporting-a-bug/) again and provide a [minimal reproduction](https://squidfunk.github.io/mkdocs-material/guides/creating-a-reproduction/) by using [the info plugin](https://squidfunk.github.io/mkdocs-material/guides/creating-a-reproduction/#creating-a-zip-file). Thank you.

**Comment by ofek** [comment]
[9.5.1+insiders.4.47.0-social-card-logo-overlap.zip](https://github.com/squidfunk/mkdocs-material/files/13628106/9.5.1%2Binsiders.4.47.0-social-card-logo-overlap.zip)

**Comment by squidfunk** [comment]
Thanks for the reproduction. So I read this only happens in Insiders?

**Comment by squidfunk** [comment]
Okay, this is still not a minimal reproduction. It contains your entire site. Please provide a minimal reproduction that only includes the minimal set of files that showcases the error. It's not runnable:

```
$ mkdocs build
ERROR   -  Config value 'markdown_extensions': Failed to load extension 'mkdocs-click'.
           ModuleNotFoundError: No module named 'mkdocs-click'
ERROR   -  Config value 'plugins': The "glightbox" plugin is not installed
ERROR   -  Config value 'watch': The path '/Users/squidfunk/Downloads/9.5.1+insiders.4.47.0-social-card-logo-overlap/backend/src/hatchling' isn't an existing file
           or directory.
```

**Comment by ofek** [comment]
I thought that was an Insiders feature but I just installed the latest release instead and actually there is no overlap but rather the logo defined has no effect whatsoever and it uses the site logo only.

![index](https://github.com/squidfunk/mkdocs-material/assets/9677399/7a1e2d63-b1a6-4867-acac-488353ab7371)

**Comment by squidfunk** [comment]
So, does this solve your problem? I'm not sure I understand your last message... If not, we definitely need a minimal reproduction. Also note that the social plugin was entirely rewritten in Insiders. It has no commonalities with the community edition.

**Comment by ofek** [comment]
Oh, indeed the option is insiders only which is why it has no effect on the latest public release https://squidfunk.github.io/mkdocs-material/plugins/social/#option.logo

Sure, I'll put together a small example. I literally just want that logo to be replaced by the path to image

**Comment by squidfunk** [comment]
Thank you.

**Comment by ofek** [comment]
[material-repro.zip](https://github.com/squidfunk/mkdocs-material/files/13628335/material-repro.zip)

**Comment by ofek** [comment]
I feel like this is such a fundamental feature and so simple that I must be doing something wrong in that example I just gave you. Do you think perhaps SVGs aren't supportive for that field and maybe I should use a different format?

**Comment by squidfunk** [comment]
Is there any reason you are not using [the info plugin as required](https://squidfunk.github.io/mkdocs-material/guides/creating-a-reproduction/#creating-a-zip-file)? The info plugin collects all installed dependencies and information on the environment that is missing in your reproduction... You know, I'm happy to help you, but we have processes for a reason. I'm feeling really sorry to be so upfront, but having to ask for each and every bit of information is a very inefficient use of my time.

**Comment by ofek** [comment]
I'm sorry! I did the info plugin originally but you tried the archive and said it was not reproducible so I had to manually take away everything. Is that not what you meant for me to do?

**Comment by squidfunk** [comment]
A minimal reproduction should always be built from scratch, [as described in our documentation](https://squidfunk.github.io/mkdocs-material/guides/creating-a-reproduction/#minimal-reproduction).

**Comment by squidfunk** [comment]
Fixed in [`4aa87aa`](https://github.com/squidfunk/mkdocs-material-insiders/commit/4aa87aab257ebbfd458fe1a36027391a1e8d96e7). It took 3min with a minimal reproduction 😉

**Comment by ofek** [comment]
I will do better with reproduction next time, thank you!

![index](https://github.com/squidfunk/mkdocs-material/assets/9677399/57241ce4-7d63-4310-b471-f6779faabd73)

**Comment by squidfunk** [comment]
Keeping open until released.

**Comment by squidfunk** [comment]
Released as part of [9.5.2+insiders-4.47.1](https://squidfunk.github.io/mkdocs-material/insiders/changelog/#4.47.1).
